src/Validation/Validation.php

```diff
@@ -38,6 +38,62 @@ class Validation
      */
     const DEFAULT_LOCALE = 'en_US';
 
+    /**
+     * Same as operator.
+     *
+     * @var string
+     */
+    const COMPARE_SAME_AS = '===';
```

Comment:
to be consistent I would ask for

- COMPARE_SAME
- COMPARE_NOT_SAME
- COMPARE_EQUAL
- COMPARE_NOT_EQUAL
- COMPARE_GREATER
- COMPARE_GREATER_OR_EQUAL
- COMPARE_LESS
- COMPARE_LESS_OR_EQUAL

Also would be most concise while still being descriptive.